packages/shared/wearablesPortableExperience/sagas.ts

```diff
@@ -109,13 +109,18 @@ function* handleStartWearablesPortableExperience(action: StartWearablesPortableE
         r.contents.some((c) => c.key.endsWith('game.js'))
       )[0].contents
 
-      // Get the game.js path
-      const gameJs = wearableContent.filter((c) => c.key.endsWith('game.js'))[0].key
-      const gameJsRootLength = gameJs.lastIndexOf('/') + 1
-
-      // Convert wearable mapping to scene's mapping and modify file path so game.js directory becomes the root directory
-      // for asset loading
-      const mappings = wearableContent.map((c) => ({ file: c.key.substring(gameJsRootLength), hash: c.hash }))
+      const femaleCrop = wearableContent.filter(($) => $.key.substr(0, 7) == 'female/').length == wearableContent.length
+      const maleCrop = wearableContent.filter(($) => $.key.substr(0, 5) == 'male/').length == wearableContent.length
+
+      let mappings: any = []
```

Comment:
can we add some comments to that female/male crop ?
Also if the uniq thing that changes in the if/else clause its the file prop can we move it to a fn ?

```ts
const getFile = (key: string) => {
  if (femaleCrop) return key.substring(7)
  if (maleCrop) return key.subString(5)
  return key
}
const mappings = wearableContent.map(
  content => ({ file: getFile(content.file), hash: content.hash })
)
```

This way mapping will have types, and also inmutable 😄